src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
The enum name 'Colors' should be singular 'Color' as each instance of an enum represents a single value.

Reply:
done